Dockerfile

```diff
@@ -2,31 +2,32 @@
 FROM python:3.6-stretch AS base
 WORKDIR /usr/src/app
 
-# install MongoDB tools
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
-RUN echo "deb http://repo.mongodb.org/apt/debian stretch/mongodb-org/3.6 main" | tee /etc/apt/sources.list.d/mongodb-org-3.6.list
-RUN apt-get update
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    mongodb-org-tools \
-    mongodb-org-shell
-
 COPY requirements.txt ./
-RUN pip install --no-cache-dir -r requirements.txt
+# install MongoDB tools
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5 && \
+    echo "deb http://repo.mongodb.org/apt/debian stretch/mongodb-org/3.6 main" | tee /etc/apt/sources.list.d/mongodb-org-3.6.list && \
+    apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y mongodb-org-tools mongodb-org-shell && \
+    pip install --no-cache-dir -r requirements.txt
```

Comment:
The idea is to use as little RUN statements as possible, to have as little image layers as possible, to reduce the resulting image size. This will make pulling and starting the image quicker.

But I'm going to see if we can use an alpine image in this case, because the image is currently 1.2GB in size.